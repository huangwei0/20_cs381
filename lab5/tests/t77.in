findall(Z, older(Z, 'Prince Henry of Wales'), X), 
print(X), 
print('\n'), 
halt.
