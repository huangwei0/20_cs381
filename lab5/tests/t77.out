[King George III,Charlotte of Mecklenburg-Strelitz,Franz: Duke of Saxe-Coburg-Saalfeld,Augusta Reuss of Ebersdorf und Lobenstein,King George IV,King William IV,Edward: Duke of Kent,Caroline of Brunswick,Ernest I: Duke of Saxe-Coburg and Gotha,Princess Victoria of Saxe-Coburg-Saalfeld,Princess Louise of Saxe-Gotha-Altenburg,Prince Albert,Queen Victoria,King Edward VII,Prince Alfred: Duke of Edinburgh,Prince Arthur,Prince Leopold,Victoria: Princess Royal,Princess Alice,John Campbell: Duke of Argyll,Princess Helena,Princess Louise,Princess Beatrice,Henry: Prince of Battenberg,Louis IV: Grand Duke of Hesse,Princess Helena of Waldeck and Pyrmont,Princess Victoria of Hesse,Prince Louis of Battenberg,Princess Louise of Prussia,Princess Alexandra of Denmark,Albert: Duke of Clarence,Maria: Grand duchess of Russia,King George V,Prince John,Princess Victoria,Louise: Princess Royal,Princess Maud,Haakon VII: King of Norway,Prince Christian of Schleswig-Holstein,Frederick III: German Emperor,Alexander Duff: Duke of Fife,Mary: Princess of Teck,King Edward VIII,King George VI,Henry: Duke of Gloucester,Prince George: Duke of Kent,Prince John of the United Kingdom,Mary: Princess Royal,Wallis Simpson,Alice: Duchess of Gloucester,Charles Lascelles: Earl of Harewood,Marina: Princess of Greece,Alice: Princess of Baltenberg,Andrew: Prince of Greece,Prince Phillip: Duke of Edinburgh,Lady Elizabeth Bowes-Lyon,Queen Elizabeth II,Princess Margaret,Lady Diana Spencer,Anne: Princess Royal,Catherine Middleton,Anthony Armstrong-Jones,Sarah Armstrong-Jones,David: Viscount Linley,Charles: Prince of Wales,Andrew: Duke of York,Mark Phillips,Edward: Earl of Wessex,Prince William: Duke of Cambridge,Sophie Rhys-Jones,Sarah Ferguson,Catherine Middleton,Peter Phillips,Zara Phillips,Autumn Kelly,Mike Tindall,Serena Armstrong-Jones,Daniel Chatto,George Lascelles,Gerald Lascelles,Marion Stein,Angela Dowding,Meghan Markle]
